### DISCUSSION TITLE
Solicitação de email para o CDC

### DISCUSSION THREAD

**Discussion by AndressaLF** [apyb/comunidade#126] [Geral]
Gostaria de saber sobre a possibilidade de criar um email oficial que possa ser usado pelo CDC para organização de documentação e aplicação dos formulários durante os eventos ou de forma definitiva.

@naanadr

**Comment by naanadr** [comment]
Olá @AndressaLF bom dia 😄 _formalizando por aqui o que conversamos durante a semana._

Já que você e a @anicelysantos estão trabalhando em um documento que visa apoiar com diretrizes, os comitês de CDC dos eventos que são apoiados pela APyB (Issue https://github.com/apyb/tarefas/issues/645). Por enquanto que não temos esse documento finalizado e disponível para consulta no nosso site, o que ficou decidido é:

1. Criação de um e-mail `nordeste.cdc@` onde vocês terão acesso a uma caixa de e-mail e ao google drive;
2. Ao se criar um formulário de denuncia, os dados da pessoa denunciante deverá ser anônimo, oferecendo dessa forma um ambiente seguro para ela;
3. Ao final do evento, os dados referentes a pessoa denunciada deverá ser anonimizado;
4. A senha do e-mail não deve ser compartilha com ninguém fora do comitê, dessa forma, o próximo comitê de CdC da Python Nordeste deverá entrar em contato com a APyB solicitando acesso ao e-mail.

**Reply by rougeth** [reply]
> Criação de um e-mail nordeste.cdc@ onde vocês terão acesso a uma caixa de e-mail e ao google drive;

Por que não usar um e-mail no domínio do site do evento? cdc@pythonnordeste.org não seria melhor?

> Ao se criar um formulário de denuncia, os dados da pessoa denunciante deverá ser anônimo, oferecendo dessa forma um ambiente seguro para ela;
>
> Ao final do evento, os dados referentes a pessoa denunciada deverá ser anonimizado;

Se o e-mail usado será o gerenciado pela APyB, como a APyB vai se certificar de que os eventos estão seguindo as normas de uso (expandindo para mais eventos, edições futuras, etc)? A diretoria tem condições de assumir esse trabalho?

Nos eventos em que eu participei como parte do time de resposta, escolhemos criar um e-mail específico para aquela edição exatamente para não correr o risco dos eventos seguintes acessarem informações sobre as denúncias.

Eu entendo que ter um email mais simples é mais atrativo, talvez criar um lista seja um meio termo mais seguro? Dessa forma a APyB poderia atualizar a lista de pessoas com acesso a lista sempre que a equipe mudar.

**Reply by naanadr** [reply]
> Por que não usar um e-mail no domínio do site do evento?

No Admin do google não apareceu a opção de customizar o domínio do e-mail, ele só oferece a opção de `python.org.br` . Você chegou a conseguir fazer e-mails a partir de subdomínios? 🤔 

> Se o e-mail usado será o gerenciado pela APyB, como a APyB vai se certificar de que os eventos estão seguindo as normas de uso (expandindo para mais eventos, edições futuras, etc)? 

Estamos debatendo isso internamente, e criando um documento com uma norma de uso do e-mail de cdc e um termo de tratamento dos dados (_parecido com o termo de big kahuna da PyBr_), que deverá ser de responsabilidade da pessoa solicitante do acesso ao e-mail. 

> Nos eventos em que eu participei como parte do time de resposta, escolhemos criar um e-mail específico para aquela edição exatamente para não correr o risco dos eventos seguintes acessarem informações sobre as denúncias.

Pensamos nisso também, por isso que no documento que será disponibilizado futuramente terá essa obrigação do comitê normalizar os dados (_algo que já deveria ser feito estando utilizando um e-mail da apyb ou não, visto que informações sensíveis são coletadas_). E além do documento com as diretrizes, também terá um termo que detalhará quais as responsabilidades de cada parte envolvida. 

> Eu entendo que ter um email mais simples é mais atrativo, talvez criar um lista seja um meio termo mais seguro? Dessa forma a APyB poderia atualizar a lista de pessoas com acesso a lista sempre que a equipe mudar.

Pensei nisso, mas o comitê tem a necessidade de ter acesso a um drive para poder fazer formulários, e armazenar o histórico de casos anteriores para tomar decisões mais assertivas. Porque já notamos que não ter conhecimento de como comitês anteriores agiram, às vezes acaba atrapalhando mais do que ajudando 😞

**Reply by rougeth** [reply]
> No Admin do google não apareceu a opção de customizar o domínio do e-mail, ele só oferece a opção de python.org.br . Você chegou a conseguir fazer e-mails a partir de subdomínios? 🤔

Precisaria adicionar o domínio da Python Nordeste no Google Admin
https://support.google.com/a/answer/7502379?hl=en. A partir daí, você vai poder criar _aliases_ com o endereço @pythonnordeste.org.
